metricbeat/module/vsphere/datastore_usage/datastore_usage_test.go

```diff
@@ -0,0 +1,10 @@
+package datastore_usage
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestFetchEventContents(t *testing.T) {
```

Comment:
I'm not sure how can I do this. I take the values already formatted from the library. I'll see if I can fake this.